Command/ActivateContentTypeCommand.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
```

Comment:
```suggestion
declare(strict_types=1);
```
For new files we should start doing this